src/components/meetupSection.js

```diff
@@ -57,83 +59,95 @@ const MeetupSection = () => (
             4px 4px 0 ${theme.colors.lightBlue},
             5px 5px 0 ${theme.colors.lightBlue}
           `,
-        }}
-      >
-        Community Meet‑ups
-      </Text>
+          }}
+        >
+          Community Meet‑ups
+        </Text>
+
+        <Text
+          as="h3"
+          fontSize={[2, 3]}
+          color="white"
+          fontWeight="heading"
+          mb={2}
+          sx={{ textTransform: "uppercase" }}
+        >
+          Past
+        </Text>
 
-      <Box
-        sx={{
-          display: "grid",
-          gridGap: "3rem 2rem",
-          gridTemplateColumns: [
-            "auto",
-            null,
-            `repeat(auto-fit, minmax(25em, 1fr))`,
-          ],
-        }}
-      >
-        {meetups.map(({ id, logo, title, date, venue, replay }) => (
+        {data.map(({ id, name, start_time, place }) => (
           <Flex
             key={id}
-            flexDirection="column"
-            width="100%"
-            maxWidth="30em"
-            color="darkblue"
-            mt={2}
-            mx="auto"
+            as="li"
+            bg="white"
+            p={2}
+            mb={2}
+            color="darkBlue"
+            alignItems={["start", "center"]}
+            flexDirection={["column", "row"]}
           >
-            <Image
-              src={logo}
-              alt={title}
-              width="12.5rem"
-              mx="auto"
-              mb={3}
-              display="block"
-            />
-
-            <Flex
-              bg="white"
-              p={2}
-              flexDirection="column"
-              flex={1}
+            <Box
+              as="time"
+              dateTime={start_time}
+              pr={2}
+              mr={2}
               sx={{
-                boxShadow: "main",
-                transition: "200ms transform",
-                "&:hover": {
-                  transform: "translateY(-0.5rem)",
-                },
+                textTransform: "uppercase",
+                borderRightWidth: "1px",
+                borderRightStyle: "solid",
+                borderRightColor: ["transparent", "darkBlue"],
               }}
             >
-              <Box flex={1}>
-                <Flex alignItems="baseline" fontWeight="bold" fontSize={2}>
-                  <Link href="/">{title}</Link>
-                  <Text ml="0.5rem">›</Text>
-                </Flex>
-                <Text my="0.25rem">{date}</Text>
-                <Text>{venue}</Text>
-              </Box>
+              {formatDate(start_time)}
+            </Box>
+
+            <Box flex={1} pt={[1, 0]} pb={[2, 0]}>
+              <Text
```

Comment:
Can you add a right margin here just so it doesn't ever push up against the button?